README.md

```diff
@@ -1,88 +1,87 @@
 # Zowe CLI Sample Plug-in
 
-This repository contains a sample Zowe CLI plug-in that adheres to the contribution guidelines for the project. Use this project and the associated tutorials as a starting point for creating Zowe CLI plug-ins.
+This repository contains a sample Zowe CLI Plug-in that adheres to the contribution guidelines for the project. Use this project and the associated tutorials as a starting point for creating Zowe CLI Plug-ins.
 
 - [Zowe CLI Sample Plug-in](#zowe-cli-sample-plug-in)
+  - [Sample Plug-in Overview](#sample-plug-in-overview)
   - [Why Create a Zowe CLI Plug-in?](#why-create-a-zowe-cli-plug-in)
   - [Tutorials, Documentation, and Guidelines](#tutorials-documentation-and-guidelines)
     - [Tutorials](#tutorials)
-    - [Contribution Guidelines](#contribution-guidelines)
     - [Imperative CLI Framework Documentation](#imperative-cli-framework-documentation)
-    - [Jenkinsfile Guidelines](#jenkinsfile-guidelines)
-  - [Prerequisites](#prerequisites)
-  - [Create a Local Development Space](#create-a-local-development-space)
-    - [Clone zowe-cli-sample-plugin and Build From Source](#clone-zowe-cli-sample-plugin-and-build-from-source)
-    - [Run the Automated Tests](#run-the-automated-tests)
-    - [Install the zowe-cli-sample-plugin to Zowe CLI](#install-the-zowe-cli-sample-plugin-to-zowe-cli)
+    - [Contribution Guidelines](#contribution-guidelines)
+    - [CICD Guidelines](#cicd-guidelines)
+  - [Set up your Development Environment](#set-up-your-development-environment)
+    - [Prerequisites](#prerequisites)
+    - [Initial Setup](#initial-setup)
+    - [Next Steps: Using your Sample Plug-in](#next-steps-using-your-sample-plug-in)
+
+## Sample Plug-in Overview
+
+This sample includes:
+- Source code for extending the Zowe CLI with a command that lists the contents of a remote directory.
+- Tutorials for:
+  - [Building the plug-in](docs/tutorials/list-directory-contents/ListDirectoryContentsPlugin.md) to use its `zowe zowe-cli-sample list directory-contents` command.
+  - [Enhancing the sample](docs/tutorials/files-util/FilesUtilPlugin.md) with a third-party package and creating a new command for file-diff highlighting.
+  - [Augmenting the code](docs/tutorials/list-typicode-todo/ListTypicodeTodoPlugin.md) to connect with Typicode's external API, a simulator of backend data, to walk through the cli's interaction with RESTful services.
+  - [Using profiles](docs/tutorials/profile-example/ProfilePlugin.md) with your plug-in.
+- Examples of lifecycle hooks and GitHub Actions workflows for CICD.
+- Templates for unit, system and integration testing suites.
 
 ## Why Create a Zowe CLI Plug-in?
 
-You might want to create a Zowe CLI plug-in to accomplish the following:
+You might want to create a Zowe CLI Plug-in to accomplish the following:
```

Comment:
There may be more instances of `Plug-in` that may need to go back to lowercase, unless we consider the word plug-in as part of the name? (@anaxceron)

Reply:
made the fix. thanks fernando!